dist/api_data.py

```diff
@@ -1518,9 +1518,17 @@ def __ge__(self, other):
         to the bit count (except for the last set of values loaded)'''),
     Config('checkpoint', '', r'''
         the name of a checkpoint to open (the reserved name
-        "WiredTigerCheckpoint" opens the most recent internal
+        "WiredTigerCheckpoint" opens the most recent
         checkpoint taken for the object).  The cursor does not
         support data modification'''),
+    Config('checkpoint_use_history', 'true', r'''
+        when opening a checkpoint cursor, open history store cursors and retrieve
+        snapshot and timestamp information from the checkpoint. This is in general
+        required for corect reads; if setting it to false the caller must ensure
+        that the checkpoint is self-contained in the data store: needed for correct
+        reads: timestamps are not in use and the object was quiescent when the
+        checkpoint was taken''',
+        type='boolean', undoc=True),
```

Comment:
two colon-separated clauses reads oddly

Reply:
Agreed. And... the middle bit is redundant and doesn't belong there at all, so I got rid of it. Must have been an edit mistake. (Fixed in c25a4ab)